test/uk/gov/hmrc/cdsreimbursementclaim/services/ClaimTransformerServiceSpec.scala

```diff
@@ -1026,16 +1025,16 @@ class ClaimTransformerServiceSpec extends AnyWordSpec with Matchers with MockFac
         val declarantType                               = DeclarantTypeAnswer.Importer
         val maybeClaimantDetailsAsImporterCompanyAnswer = None
 
-        val acc14                                                            = getAcc14Response()
-        val displayDeclaration                                               = sample[DisplayDeclaration].copy(displayResponseDetail = acc14)
-        val claim                                                            = getClaimAmounts()
-        val claimsAnswer                                                     = ClaimsAnswer(claim)
-        val completeBankAccountDetailAnswer: CompleteBankAccountDetailAnswer = sample[CompleteBankAccountDetailAnswer]
-        val basisOfClaimAnswer                                               = BasisOfClaim.DutySuspension
-        val declarantTypeAnswer                                              = declarantType
-        val completeMovementReferenceNumberAnswer                            = sample[MovementReferenceNumber]
+        val acc14                                               = getAcc14Response()
+        val displayDeclaration                                  = sample[DisplayDeclaration].copy(displayResponseDetail = acc14)
+        val claim                                               = getClaimAmounts()
+        val claimsAnswer                                        = ClaimsAnswer(claim)
+        val completeBankAccountDetailAnswer: BankAccountDetails = sample[BankAccountDetails]
+        val basisOfClaimAnswer                                  = BasisOfClaim.DutySuspension
+        val declarantTypeAnswer                                 = declarantType
+        val completeMovementReferenceNumberAnswer               = sample[MovementReferenceNumber]
```

Comment:
Perhaps rename `completeMovementReferenceNumberAnswer` to `movementReferenceNumberAnswer`